packages/mongodb-rag-core/src/dataSources/MdOnGithubDataSource.test.ts

```diff
@@ -35,13 +36,13 @@ const mongodbCorpConfig: MakeMdOnGithubDataSourceParams = {
   extractTitle: (_, frontmatter) => (frontmatter?.title as string) ?? null,
 };
 
-const ingestTestDataConfig: MakeMdOnGithubDataSourceParams = {
+const testDataConfig: MakeMdOnGithubDataSourceParams = {
   ...baseChatbotRepoConfig,
   name: "ingest_testData",
   metadata: {
     productName: "Ingest Test Data",
   },
-  filter: (path) => path.includes("ingest/testData"),
+  filter: (path) => path.includes(Path.join("mongodb-rag-core", "testData")),
```

Comment:
Do we need the `Path` import here? Can't you just do `"mongodb-rag-core/testData"` since this is joining two static strings?

Reply:
not if you're on windows!